src/main/java/com/conveyal/gtfs/graphql/fetchers/JDBCFetcher.java

```diff
@@ -99,9 +114,38 @@ public static GraphQLFieldDefinition field (String tableName) {
         // This DataFetcher only makes sense when the enclosing parent object is a feed or something in a feed.
         // So it should always be represented as a map with a namespace key.
 
-        // GetSource is the context in which this this DataFetcher has been created, in this case a map representing the parent feed.
-        Map<String, Object> parentEntityMap = environment.getSource();
         String namespace = (String) parentEntityMap.get("namespace");
+
+        List<String> parentJoinValues = new ArrayList<>();
+        if (parentJoinField != null) {
```

Comment:
The method has been refactored so that the fetching action can be used by NestedJdbcFetcher as well.  The comment you're describing has just been moved below, but I will add in an extra comment here.